paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPluginExtension.kt

```diff
@@ -0,0 +1,16 @@
+package app.cash.paparazzi.gradle
+
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.file.ProjectLayout
+import org.gradle.api.model.ObjectFactory
+import javax.inject.Inject
+
+open class PaparazziPluginExtension
+@Inject constructor(
+  objectsFactory: ObjectFactory,
+  layout: ProjectLayout
+) {
+
+  val snapshotDirectory: DirectoryProperty =
+    objectsFactory.directoryProperty().convention(layout.projectDirectory.dir("src/test/snapshots"))
```

Comment:
I've asked Gradle explicitly about this in the past, they don't recommend change to:

```suggestion
  abstract val snapshotDirectory: DirectoryProperty
```
\+
```
val extension = project.extensions.create("paparazzi", PaparazziPluginExtension::class.java) {
    snapshotDirectory.convention(project.layout.projectDirectory.dir("src/test/snapshots"))
}
```

Reply:
Done

Reply:
resolve